src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -1192,6 +1192,127 @@ public async Task ReinstateSubscriptionAsync(ISubscriber subscriber)
         }
     }
 
+    public async Task RemovePaymentMethod(Organization organization)
```

Comment:
⛏️ Since this file is already so large, I'd recommend against extending it further. It doesn't have to derail this PR, but do think about moving this to its own command.